bin/parseDocs.ts

```diff
@@ -194,4 +194,28 @@ for (const key in json.recipes) {
 	}
 }
 
+// attach extractable resources instead of keeping empty array with "everything allowed"
+for (const minerKey in json.miners) {
+	if (json.miners[minerKey].allowedResources.length > 0) {
+		continue;
+	}
+
+	const allowedResources = [] as string[];
+	for (const resourceKey in json.resources) {
+		if (!json.items[resourceKey]) {
+			throw new Error(`Item of resource type "${resourceKey}" was not found.`);
+		}
+
+		const item = json.items[resourceKey];
+
+		if (item.liquid !== json.miners[minerKey].allowLiquids) {
```

Comment:
if think more readable would be `if (item.liquid === json.miners[minerKey].allowLiquids) { allowedResources.push(resourceKey) }`, so you won't have to do the `continue` thingy

Reply:
Actually that notation act's like simple NoSQL database, which is very quick, other thing is that, whole `json.miners` or even whole `json` is already in memory, so impact is very low,

Reply:
> if think more readable would be `if (item.liquid === json.miners[minerKey].allowLiquids) { allowedResources.push(resourceKey) }`, so you won't have to do the `continue` thingy

Yes, that's right, fix in progress